tests/server/lang.js

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Mostly intended to detect possible failures in packaging production builds,
+// but should be expected to pass against latest.dev.lcip.org as well.
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'intern/dojo/node!../../server/lib/configuration',
+  'intern/dojo/node!request',
+  'intern/dojo/node!url',
+  'intern/dojo/node!../../server/config/production-locales',
+], function (intern, registerSuite, assert, config, request, url, productionLocales) {
+
+  var languages = productionLocales.i18n.supportedLanguages;
+  var httpsUrl = intern.config.fxaContentRoot.replace(/\/$/, '');
+
+  var suite = {
+    name: 'load / with various accept-languages'
+  };
+
+  function langTest(lang) {
+    var options = {
+      headers: {
+        'Accept-Language': lang
+      }
+    };
+
+    suite['#https get ' + httpsUrl + '/ -> ' + lang] = function () {
+      var dfd = this.async(intern.config.asyncTimeout);
+      request(httpsUrl + '/', options, dfd.callback(function (err, res) {
+        assert.ok(!err);
+        assert.equal(res.statusCode, 200);
```

Comment:
Should we do a check here that the language attribute shows up correctly on the HTML tag? E.g. `      assert.ok(res.body.match(/lang="zh-CN"/));`.